### ISSUE TITLE
Inconsistent login state depending on www

### ISSUE THREAD

**Issue by equin0x80** [demovibes/demovibes#16]
If you log in at scenestream.net (no www) and then visit a page with the www, such as:
https://www.scenestream.net/demovibes/tags/EarlyMOD/
...then you are not shown as logged in on that page, and e.g. cannot queue songs from that page.

**Comment by greg-kennedy** [comment]
This seems like an nginx configuration error.  Ideally we should have one canonical URL (www or non-www), and attempts to browse to the wrong are 301 redirected to the correct name.

**Comment by equin0x80** [comment]
Yes, I now realise this is a generic server issue and probably doesn't belong on the demovibes issue list. I'm not really sure where else to report that type of thing, except your or Phlubby's inbox :)
